src/js/date.js

```diff
@@ -104,51 +107,37 @@ export const getLocaleDayMonthYear = (locale = 'en-uk', date = new Date()) => {
 };
 
 export const isDateValid = (locale = 'en-uk', inputValue = '') => {
-  let splittedInput = [];
-  let orderedInput = [];
-  let date = null;
-
-  if (locale === 'de-ch') {
-    splittedInput = inputValue.split('.');
-    orderedInput = [splittedInput[1], splittedInput[0], splittedInput[2]];
-    date = new Date(orderedInput.join('.'));
-  } else {
-    date = new Date(inputValue);
-  }
+  // year, monthIndex, day
+  const blueprint = new Date(2017, 2, 23);
```

Comment:
why not `const blueprint = new Date();`?

and then on row 133 this:

`const monthIndex = splittedBlueprint.indexOf(blueprint.getMonth());
  const dayIndex = splittedBlueprint.indexOf(blueprint.getDate());
  const yearIndex = splittedBlueprint.indexOf(blueprint.getFullYear());`